tuf-notary/registry-access.go

```diff
@@ -0,0 +1,56 @@
+package tufnotary
+
+import (
+	"context"
+	"io/ioutil"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+
+    "oras.land/oras-go/pkg/content"
+    "oras.land/oras-go/pkg/oras"
+)
+
+func UploadTUFMetadata(registry string, repository string, name string, reference string) (ocispec.Descriptor, error) {
+	ref := registry + "/" + repository + ":" + name
+	fileName := repository + "/staged/" + name + ".json"
+	contents, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return ocispec.Descriptor{}, err
+	}
+
+	fileContent := []byte(contents)
+	mediaType := "application/vnd.cncf.notary.tuf+json"
+
+	ctx := context.Background()
+
+	// TODO: add reference once it's supported in oras-go: https://github.com/oras-project/oras-go/pull/35
+
+	memoryStore := content.NewMemory()
```

Comment:
Nit: indentation kinda funny... would you mind running the source code against `gofmt` in an automated GitHub check going fwd? Also, using an IDE like VScode or Goland will go a long way.